packages/web-vite/src/lib/anchoredHighlights.ts

```
@@ -0,0 +1,457 @@
+  })
+}
+
+/**********************
+ * Hook: useAnchors   *
+ **********************/
+
+export function useAnchoredHighlights(
+  contentRef: React.RefObject<HTMLElement>,
+  highlights: AnchoredHighlight[]
+) {
+  const reapply = useRef<() => void>(() => {})
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)